llvm/lib/Target/X86/X86ISelDAGToDAG.cpp

```diff
@@ -6380,18 +6386,20 @@ void X86DAGToDAGISel::Select(SDNode *Node) {
     switch (Node->getOpcode()) {
     default:
       llvm_unreachable("Unexpected opcode!");
+#define GET_EGPR_IF_ENABLED(OPC) (Subtarget->hasEGPR() ? OPC##_EVEX : OPC)
```

Comment:
We are in the same file, really don't need to undef a macro and define it again.